java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1249,6 +1249,41 @@ public void zrank(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zremrangebyscore(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("one", 1.0, "two", 2.0, "three", 3.0, "four", 4.0);
+        assertEquals(4, client.zadd(key1, membersScores).get());
+
+        assertEquals(
+                2, client.zremrangebyscore(key1, new ScoreBoundary(1, false), new ScoreBoundary(3)).get());
```

Comment:
Can we rename `ScoreBoundary` to `ScoreBound` to be consistent with `InfScoreBound`?

Reply:
Sure do you want that in another PR?

Reply:
You can do this here.

Reply:
I'll do it another PR will be much cleaner for review.